datahub-web-react/src/app/entity/dataset/shared/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { DatasetProperties, Operation } from '../../../../types.generated';
+
+export function getLastUpdatedMs(
+    properties: Pick<DatasetProperties, 'lastModified'> | null | undefined,
+    operations: Pick<Operation, 'lastUpdatedTimestamp'>[] | null | undefined,
+): number | undefined {
+    return (
+        Math.max(
+            properties?.lastModified?.time || 0,
+            (operations?.length && operations[0].lastUpdatedTimestamp) || 0,
```

Comment:
let's put this logic in the graphql layer

Reply:
Talked to Chris, looks like it's pretty difficult to combine info from non-timeseries and timeseries aspects, so going to keep this part as is, unless you have a suggested way around that